react-components/src/components/RevealToolbar/ModelsList.tsx

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type ReactElement } from 'react';
+
+import { Menu } from '@cognite/cogs.js';
+import { type ModelWithRevision } from '../../hooks/types';
+
+export type ModelListProps = {
```

Comment:
I do not see the need to export the `type`

Reply:
https://github.com/cognitedata/reveal/pull/4746/commits/f51740e72efab7b49df11e4dc584ae056644e8f9